.github/workflows/linux_build.yml

```diff
@@ -269,6 +276,13 @@ jobs:
             TEST_CMD="ctest -V -j $(nproc)"
           fi
 
+          if test "${{ matrix.id }}" = "benchmarks"; then
```

Comment:
Move into `test.sh` ?

Reply:
I don't think this can be done under Docker. Anyway it doesn't look like this is even hit (perhaps because we already run in a container/VM that doesn't expose turboboost setting)